src/frontend/src/lib/styles/global/gix.scss

```diff
@@ -15,7 +15,7 @@
 	--value-color: var(--color-dark);
 
 	--border-radius-2x: var(--border-radius-lg);
-	--border-radius-0_5x: calc(0.5 * var(--border-radius-sm));
+	--border-radius-0_5x: var(--border-radius-xs);
```

Comment:
This change impacts the overall app - i.e. other components that the modal, such as tags, progress steps etc. This is expected?

Reply:
that is just a small simplification that i put there:

`--border-radius-xs` by definition is `calc(0.5 * var(--border-radius-sm))`